image/image.go

```diff
@@ -119,13 +109,18 @@ func (i *genericImage) DockerTar() ([]byte, error) {
 // will support v1 one day...
 type genericManifest interface {
 	String() string
-	GetLayers() []string
+	Layers() []string
+	Config() ([]byte, error)
+	ImageInspectInfo() (*types.ImageInspectInfo, error)
```

Comment:
yup, Config returns the raw config which is needed for system containers for instance iirc